discovery.sh

```diff
@@ -2,10 +2,15 @@
 #
 # discovery.sh
 #
+vinLast=""
 
-setup_auto_discovery() {
+function setupHAAutoDiscoveryEnvVars() {
   vin=$1
-  log_notice "Setting up HA auto discovery for vin $vin"
+
+  log_debug "Entering setupHAAutoDiscoveryEnvVars() vin:$vin"
+
+  [ "$vinLast" == "$vin" ] && return
+  vinLast=$1
```

Comment:
There is a hardcoded SW_VERSION below. Maybe move it to top of run.sh?

Reply:
I created version.sh but ... quickly, why not put the CHANGELOG in the Docker image? This way at start the script can grab the version directly from the CHANGELOG - and might not a bad idea to have the CHANGELOG pushed out w/ the app itself.

We can continue to think about this one... perhaps the Dockerfile can just run a command to grab the version and push it around - OR ... actually earlier today, I thought of putting that in the pre-commit and all versions would be in-sync from what's in the CHANGELOG.

Reply:
I think pre-commit in stage/main could make sense yes.
However, we need somewhere a kind of single referential like VERSION.md with sections (in core?).

I will think about the format, but most likely a kind of table where:
- 1 line is a change
- 3 columns: core, docker, addon
- value inside the cells is the individual version number in which this change has been integrated.
For an addon only change, then the box for core is grayed out or has the "n/a" value (not applicable)